ocaml/xenopsd/lib/suspend_image.ml

```diff
@@ -62,7 +62,8 @@ type header_type =
   | Qemu_xen
   | Demu
   | Varstored
-  | Swtpm
+  | Swtpm_old
```

Comment:
I would use something like `Swtpm_pre_<version>` or `Swtpm0` - just in case you need another increment.